src/physics/em/detail/RayleighData.hh

```diff
@@ -0,0 +1,240 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file RayleighData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Types.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!                                                                          
+ * Tabulated data of  modified fit formulas from Dermott E. Cullen, Nucl. 
+ * Instrum. Meth. Phys. Res. B v.101, (4),499-510.
+ *
+ * Excerpted from from G4RayleighAngularGenerator.cc of Geant4 6.10
```

Comment:
Can you also add a comment about what the units are?

Reply:
Added more description.